src/components/table-transactions/table-transactions.html

```diff
@@ -0,0 +1,43 @@
+<table class="table-transactions">
+  <thead class="table-transactions__thead">
+    <tr class="table-transactions__tr">
+      <th class="table-transactions__th">Date</th>
+      <th class="table-transactions__th">Amount</th>
+      <th class="table-transactions__th">Description</th>
+      <th class="table-transactions__th">Tags</th>
+      <th class="table-transactions__th">Account</th>
+    </tr>
+  </thead>
+  <tbody class="table-transaction__tbody">
+    <tr class="table-transactions__tr">
+      <td class="table-transactions__td">12 Mar</td>
+      <td class="table-transactions__td">+ 5,742.29</td>
+      <td class="table-transactions__td">Google Paycheck</td>
+      <td class="table-transactions__td">Salary</td>
+      <td class="table-transactions__td">
+        <span class="table-transactions__span">BoA Checking</span>
+        <div class="table-transactions__three-dots"></div>
+      </td>
+    </tr>
+    <tr class="table-transactions__tr">
+      <td class="table-transactions__td">9 Mar</td>
+      <td class="table-transactions__td">- 1,400.00</td>
+      <td class="table-transactions__td">Rent</td>
+      <td class="table-transactions__td">Rent</td>
+      <td class="table-transactions__td">
+        <span class="table-transactions__span">Chase credit card</span>
+        <div class="table-transactions__three-dots"></div>
+      </td>
+    </tr>
+    <tr class="table-transactions__tr">
+      <td class="table-transactions__td">25 Feb</td>
+      <td class="table-transactions__td">- 1,420.74</td>
+      <td class="table-transactions__td">Birch Creek Apartments</td>
+      <td class="table-transactions__td">Rent</td>
+      <td class="table-transactions__td">
+        <span class="table-transactions__span">Amex Credit Card</span>
+        <div class="table-transactions__three-dots"></div>
```

Comment:
three dots should be a button -- so you can click on it. You should also use three dots icon from material components